Resources/Textures/Objects/Weapons/Guns/Launchers/pirate_cannon.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "discord:brainfood#7460 github: brainfood1183.",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+	{
+		"name": "icon"
+	},
+	{
+		"name": "inhand-left",
+		"directions": 4
+	},
+	{
+		"name": "inhand-right",
+		"directions": 4
+	},
+    {
+      "name": "equipped-BACKPACK",
+      "directions": 4
+    }
+    ]
+}
```

Comment:
Please don't mix spaces and tabs like this, configure your IDE to just convert it all to spaces.